libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h

```diff
@@ -119,32 +139,26 @@
 #define pkcs11RSA_SIGNATURE_INPUT_LENGTH     51UL
 
 /**
+ * @ingroup pkcs11_wrapper_macros
  * @brief Elliptic-curve object identifiers.
  * From https://tools.ietf.org/html/rfc6637#section-11.
  */
 #define pkcs11ELLIPTIC_CURVE_NISTP256        "1.2.840.10045.3.1.7"
 
 /**
+ * @ingroup pkcs11_wrapper_macros
  * @brief Maximum length of storage for PKCS #11 label, in bytes.
  */
 #define pkcs11MAX_LABEL_LENGTH               32UL /* 31 characters + 1 null terminator. */
 
 /**
+ * @ingroup pkcs11_wrapper_macros
  * @brief OID for curve P-256.
  */
 #define pkcs11DER_ENCODED_OID_P256           { 0x06, 0x08, 0x2a, 0x86, 0x48, 0xce, 0x3d, 0x03, 0x01, 0x07 }
 
-
-/**
- * @brief Set to 1 if importing private keys is supported.
- *
- * If private key import is not supported, this value should be defined 0 in aws_pkcs11_config.h
- */
-#ifndef pkcs11configIMPORT_PRIVATE_KEYS_SUPPORTED
```

Comment:
Seems the default is changed now. Is it intentional?

Reply:
Yes, I felt like this might lead to some non-obvious behavior. Also this configuration is usually created when doing a qualification and not really touched afterwards, so it's being in the porting guide should be enough

Reply:
Going to revert it for now, and separate it into a different PR, so this one doesn't have any logic changes